cumulus/test/runtime/src/lib.rs

```diff
@@ -367,7 +364,10 @@ pub struct TestOnRuntimeUpgrade;
 
 impl OnRuntimeUpgrade for TestOnRuntimeUpgrade {
 	fn on_runtime_upgrade() -> frame_support::weights::Weight {
-		assert_eq!(sp_io::storage::get(TEST_RUNTIME_UPGRADE_KEY), Some(vec![1, 2, 3, 4].into()));
+		assert_eq!(
+			sp_io::storage::get(test_pallet::TEST_RUNTIME_UPGRADE_KEY),
+			Some(vec![1, 2, 3, 4].into())
+		);
```

Comment:
@bkchr do we still need it?

Reply:
I assume there was a bug. I have gone back in history to find out why I introduced this, but I didn't find anything :see_no_evil:

Reply:
I'd remove it, as it makes little sense. Has nothing to do with runtime upgrade.

Would make sense if upgrade would wipe the storage.

Reply:
Let's remove it. IDK. I should have left better comments...

Reply:
OK, you convinced me, I have no idea. It won't hurt us. And the assert is correct. Let it be here.